robottelo/cli/bootdisk.py

```diff
@@ -0,0 +1,41 @@
+"""
+Usage::
+
+    hammer bootdisk [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters::
+
+    SUBCOMMAND                    Subcommand
+    [ARG] ...                     Subcommand arguments
+
+Subcommands::
+
+    generic                       Download generic image
+    host                          Download host image
+    subnet                        Download subnet generic image
+"""
+from robottelo.cli.base import Base
+
+
+class Bootdisk(Base):
+    """Manipulates Bootdisk."""
+
+    command_base = 'bootdisk'
+
+    @classmethod
+    def generic(cls, options=None):
+        """Associate a resource"""
+        cls.command_sub = 'generic'
+        return cls.execute(cls._construct_command(options), output_format='json')
+
+    @classmethod
+    def host(cls, options=None):
+        """Add subscription"""
```

Comment:
Is this the correct description of the method?

Reply:
Updated